### PR TITLE
Add lighthouse scripts

### PR DESCRIPTION
**Pull request by tellyworth**
This installs `lighthouse-ci` and adds an easy script for running Lighthouse.

Props <username>, <username>

### Screenshots

Screen Shot 2022-07-28 at 12 16 28 pm

### How to test the changes in this Pull Request:

1. `yarn`
2. `yarn lighthouse`
3. `yarn lighthouse:desktop`
4. `yarn lighthouse:mobile`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
 		"update:tools": "composer update && TEXTDOMAIN=wporg composer exec update-configs",
 		"wp-env": "wp-env",
 		"backstop:reference": "npm exec backstop reference",
-		"backstop:test": "npm exec backstop test"
+		"backstop:test": "npm exec backstop test",
+		"lighthouse": "npm exec -- lighthouse-ci http://localhost:8888/ --accessibility=100 --best-practices=100",
+		"lighthouse:desktop": "npm exec -- lighthouse http://localhost:8888/ --view --preset=desktop --output-path=lighthouse.html",
+		"lighthouse:mobile": "npm exec -- lighthouse http://localhost:8888/ --view --preset=mobile --output-path=lighthouse.html"
```
Comment (ryelle) [inline package.json]:
```suggestion
		"lighthouse": "lighthouse-ci http://localhost:8888/ --accessibility=100 --best-practices=100",
		"lighthouse:desktop": "lighthouse http://localhost:8888/ --view --preset=desktop --output-path=lighthouse.html",
		"lighthouse:mobile": "lighthouse http://localhost:8888/ --view --preset=mobile --output-path=lighthouse.html"
```

**Review by ryelle — APPROVED** [review]
Works well! Just needs a rebase to fix the conflicts, and the `npm` commands aren't needed in the package.json scripts.

**Comment by StevenDufresne** [comment]
If we wanted, we could also run the tests on PR or trunk merge and save the artifact within the action.

**Comment by tellyworth** [comment]
Yeah I think lighthouse-ci will be useful for that. It's probably not needed at this stage but I might make a draft PR after merging this part.

**Pull request merged**